Tests/js/views/fields/assets/ez-relation-editview-tests.js

```diff
@@ -0,0 +1,424 @@
+            this.view.set('loadingError', this.loadingError);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        _testAvailableVariables: function (required, expectRequired) {
+            var fieldDefinition = this._getFieldDefinition(required),
+                that = this;
+
+            this.view.set('fieldDefinition', fieldDefinition);
+
+            this.view.template = function (variables) {
+                Y.Assert.isObject(variables, "The template should receive some variables");
+                Y.Assert.areEqual(9, Y.Object.keys(variables).length, "The template should receive 9 variables");
+                Y.Assert.areSame(
+                    that.jsonContent, variables.content,
+                    "The content should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.jsonVersion, variables.version,
+                    "The version should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.jsonContentType, variables.contentType,
+                    "The contentType should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    fieldDefinition, variables.fieldDefinition,
+                    "The fieldDefinition should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.field, variables.field,
+                    "The field should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.destinationContentToJSON, variables.destinationContent,
+                    "The field should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    that.view.get('loadingError'), variables.loadingError,
+                    "The field should be available in the field edit view template"
+                );
+                Y.Assert.areSame(
+                    !that.field.fieldValue.destinationContentId, variables.isEmpty,
+                    "The field should be available in the field edit view template"
+                );
+
+
+                Y.Assert.areSame(expectRequired, variables.isRequired);
+                return '';
+            };
+            this.view.render();
+        },
+
+        "Test not required field": function () {
+            this._testAvailableVariables(false, false);
+        },
+
+        "Test required field": function () {
+            this._testAvailableVariables(true, true);
+        },
+
+        "Test validate no constraints": function () {
+            var fieldDefinition = this._getFieldDefinition(false);
+
+            this.view.set('fieldDefinition', fieldDefinition);
+            this.view._set('destinationContentId', null);
```

Comment:
now that we added a `destinationContentChange` event handler, you should set the `destinationContent` attribute to avoid using the protected `_set`.